README.md

```diff
@@ -10,11 +10,13 @@ Like `npm run watch-js & npm run watch-less` but better.
 ![](docs/demo.gif)
 
 **Table of contents**
-- [Why](#why)
-- [Install](#install)
-- [Usage](#usage)
-- [Programmatic Usage](#programmatic-usage)
-- [FAQ](#faq)
+- [Concurrently](#concurrently)
+  - [Why](#why)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Programmatic Usage](#programmatic-usage)
+    - [`concurrently(commands[, options])`](#concurrentlycommands-options)
+  - [FAQ](#faq)
```

Comment:
Can you please undo the changes to this TOC?

Reply:
Yea, I didn't notice it at first. Must've been done by one of my VS Code extensions. Will revert.